chart/hyrax/README.md

```diff
@@ -90,12 +90,58 @@ For those interested in trying out or contributing to this Chart, it's helpful
 to setup a simple cluster locally. Various projects exist to make this easy; we
 recommend [`k3d`][k3d] or [minikube][minikube].
 
-For example, with `k3d`:
+### Getting Started with K3D
+
+Make sure you have `docker`, `k3d`, and `helm` installed (e.g., for OSX run `brew install k3d` and `brew install helm`; and install `Docker Desktop`).
+
+Make sure you have Docker up and running.
+
+In the Hyrax directory, run the following command; it will pull images and when finished you'll have a Docker container runninig.
```

Comment:
the directory for this doesn't matter, so this can be:

```suggestion
Run the following command; it will pull images and when finished you'll have a Docker container runninig.
```